src/components/Onboarding/steps/Init.js

```diff
@@ -117,12 +117,12 @@ const InitCardContainer = styled(Box).attrs({
   horizontal: true,
   borderRadius: '4px',
 })`
+  cursor: initial;
```

Comment:
already default

Reply:
same here, if no default we have a cursor: text